packages/api/src/command/webhook/webhook.ts

```diff
@@ -186,36 +177,39 @@ export const sendTestPayload = async (url: string, key: string): Promise<boolean
   return false;
 };
 
+// TODO move this to a separate file, this file should be as independent from MAPI/DAPI as possible.
```

Comment:
@RamilGaripov I left this as a comment and didn't move the function to minimize conflicts on your end. It would be good if you could move it to `devices.ts` and remove this comment.